docs/install.rst

```diff
@@ -108,18 +108,26 @@ To update parameters for ``local.localhost``::
     mkdir $HOME/.radical/pilot/configs/
     cp $VIRTUAL_ENV/lib/python3*/site-packages/radical/pilot/configs/resource_local.json $HOME/.radical/pilot/configs/
 
-Then edit the ``localhost`` JSON object in ``$HOME/.radical/pilot/configs/resource_local.json``.
+Then edit the ``localhost`` JSON object in :file:`$HOME/.radical/pilot/configs/resource_local.json`.
 
 More notes on Python virtual environments
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-RP recommends (but does not require) a completely static set of virtual environments.
-For simplicity and convenience, the built-in resource definitions have automatic
-environment bootstrapping logic.
+Pilot environment
+~~~~~~~~~~~~~~~~~
+
+The RADICAL Pilot remote software components are based in a Python virtual environment
+determined by parameters in the
+`resource <https://radicalpilot.readthedocs.io/en/stable/machconf.html>`__ definition.
+
+By default, RADICAL Pilot resources are configured to bootstrap the target environment
+by creating a fresh virtual environment. (``virtenv_mode=create`` and ``rp_version=local``
+in most `resource <https://radicalpilot.readthedocs.io/en/stable/machconf.html>`__ definitions.)
```

Comment:
```suggestion
in most `resource <https://radicalpilot.readthedocs.io/en/stable/machconf.html>`__ definitions.)
``virtenv_mode=update`` is a better choice than ``create``, so that later sessions can re-use a previously bootstrapped pilot venv.
```